.github/workflows/build-images.yml

```diff
@@ -30,7 +30,11 @@ jobs:
           IMAGE_NAME=$ECR_REGISTRY/$ECR_REPOSITORY
           docker build -t $IMAGE_NAME:$IMAGE_TAG . -f ci/Dockerfile --target base
           docker tag $IMAGE_NAME:$IMAGE_TAG $IMAGE_NAME:latest
-          docker push --all-tags $IMAGE_NAME
+          echo "IMAGE_NAME=$IMAGE_NAME" >> $GITHUB_ENV
+      - name: push squidex
+        if: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
Why does this only happen on workflow dispatch? What happens to the images on master?